### PR TITLE
Return bits from Number128::into_bits

### PR DESCRIPTION
**Pull request by nevi-me**

### CODE REVIEW

**Comment by suscd** [comment]
also needs a version bump

File: math/src/number_128.rs [hunk 1]
```diff
@@ -72,15 +72,33 @@ impl Number128 {
...
-    /// Get the underlying 128-bit representation
-    pub fn into_bits(self) -> i128 {
-        self.0
+    /// Get the underlying 128-bit representation in bytes.
+    /// Uses the target endianness of the caller
+    pub fn into_bits(self) -> [u8; 16] {
+        #[cfg(target_endian = "little")]
+        return self.0.to_le_bytes();
+        #[cfg(target_endian = "big")]
+        self.0.to_be_bytes()
```
Comment (suscd) [inline math/src/number_128.rs:81]: couldn't you just use `to_ne_bytes` ?

Comment (callensm) [reply math/src/number_128.rs:81]: i think most of the solana runtime uses little endian, if that matters.

File: math/src/number_128.rs [hunk 2]
```diff
@@ -72,15 +72,33 @@ impl Number128 {
...
     }
 
     /// Read a number from a raw 128-bit representation, which was previously
     /// returned by a call to `into_bits`.
-    pub fn from_bits(bits: i128) -> Self {
-        Self(bits)
+    /// Uses the target endianness of the caller
+    pub fn from_bits(bits: [u8; 16]) -> Self {
+        #[cfg(target_endian = "little")]
+        return Self(i128::from_le_bytes(bits));
```
Comment (callensm) [inline math/src/number_128.rs:89]: shouldn't need `return` for these first `cfg` conditions since its compile time

Comment (callensm) [reply math/src/number_128.rs]: oh interesting...fair enough!

**Review by suscd — APPROVED** [review]

**Pull request merged**